src/assets/translations/szit.json

```diff
@@ -14166,7 +14166,7 @@
                   "verse": "12"
                 },
                 {
-                  "text": "Ti adjatok nekik enni” – válaszolta. „Nincs, csak öt kenyerünk és két halunk – mondták. – El kellene mennünk, hogy ennyi népnek ennivalót vegyünk.”",
+                  "text": "„Ti adjatok nekik enni” – válaszolta. „Nincs, csak öt kenyerünk és két halunk – mondták. – El kellene mennünk, hogy ennyi népnek ennivalót vegyünk.”",
```

Comment:
This is **not** missing in the text edition.
And also presented in the szentiras.hu API answer:
https://szentiras.hu/api/ref/Lk9,13/SZIT

This seemt to be a synopticus.org error.

Reply:
No action needed.